Gemfile

```diff
@@ -84,9 +82,13 @@ group :test do
   gem 'rack_session_access'
   gem 'rack-test'
   gem 'rspec-activejob'
-  gem 'rubocop'
   gem 'shoulda-matchers', '~> 2.8', require: false
   gem 'sms-spec', git: 'https://github.com/monfresh/sms-spec.git', require: 'sms_spec'
   gem 'timecop'
   gem 'webmock'
 end
+
+group :linters do
```

Comment:
I just thought it would be nice to have the organized separately, but I just realized that we'll need to add yet another group to ignore when deploying, so perhaps I should keep them in development.